pymatbridge/tests/test_magic.py

```diff
@@ -82,7 +82,7 @@ def test_matrix(self):
     # Matlab struct type should be converted to a Python dict
     def test_struct(self):
         self.ip.run_cell('num = 2.567')
-        self.ip.run_cell('num_array = np.array([1.2,3.4,5.6])')
+        self.ip.run_cell('num_array = np.array([1.2,3.4,5.6]).reshape(3,1)')
```

Comment:
IIRC, it was getting returned as a (1,3) otherwise, I'll double-check and maybe fix it instead of papering around it  :)